integration/kubernetes/helm/alluxio/templates/alluxio-master.yaml

```diff
@@ -58,16 +58,19 @@ apiVersion: apps/v1
 kind: StatefulSet
 metadata:
   name: {{ $masterName }}
+  labels:
+    app: alluxio
 spec:
   selector:
     matchLabels:
-      app: {{ $masterName }}
+      name: {{ $masterName }}
```

Comment:
why did we need this change?

Reply:
Before we use `labels.app: {{ $masterName }}` for each StatefulSet. Now we use `labels.app: alluxio` for all resources so switched to use `labels.name` field as the selector.